bundles/org.openhab.binding.http/src/test/resources/simplelogger.properties

```diff
@@ -1,2 +1,2 @@
 # to run through all code-branches
-org.slf4j.simpleLogger.log.org.openhab.binding.http=trace
+org.slf4j.simpleLogger.log.org.openhab.addon.http=trace
```

Comment:
Not sure about this one.

Reply:
This should not have been changed, correct. But why is this file there?